src/white.jl

```diff
@@ -0,0 +1,110 @@
+# white.jl
+# White and Breusch-Pagan tests for heteroskedasticity
+#
+# Copyright (C) 2020   Paul Söderlind
+#
+# Permission is hereby granted, free of charge, to any person obtaining
+# a copy of this software and associated documentation files (the
+# "Software"), to deal in the Software without restriction, including
+# without limitation the rights to use, copy, modify, merge, publish,
+# distribute, sublicense, and/or sell copies of the Software, and to
+# permit persons to whom the Software is furnished to do so, subject to
+# the following conditions:
+#
+# The above copyright notice and this permission notice shall be
+# included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+# LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+# OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+# WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+export WhiteTest, BreuschPaganTest
+
+struct WhiteTest <: HypothesisTest
+    dof::Int                 #degrees of freedom in test
+    LM::Float64              #test statistic, distributed as Chisq(dof)
+    TestDescription::String  #short description of the test
```

Comment:
```suggestion
    dof::Int       # degrees of freedom in test
    lm::Float64    # test statistic, distributed as Chisq(dof)
    descr::String  # short description of the test
```

Reply:
Oh, and what does `LM` stand for?

Reply:
I thought LM = Lagrange Multiplier test statistic? 
(the only other context is LM = Linear Model)

Reply:
yes, Lagrange Multiplier test statistic. Perhaps we could use `LMstat` or `lmstat` instead

Reply:
I looked around in the lit: and it's LM everywhere. I'll stick to that.

Reply:
Though better use lowercase `lm`, in Julia uppercase is used for types.